### PR TITLE
cmd/controlclient: wire up PingRequest peerapi pings too

### PR DESCRIPTION
**Pull request by bradfitz**
Updates tailscale/corp#754

### CODE REVIEW

File: control/controlclient/direct.go [hunk 1]
```diff
@@ -1417,10 +1414,19 @@ func doPingerPing(logf logger.Logf, c *http.Client, pr *tailcfg.PingRequest, pin
...
-	})
+
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+
+	res, err := pinger.Ping(ctx, pr.IP, pingType)
+	if err != nil {
+		if ctx.Err() != nil {
+			return
+		}
```
Comment (maisem) [inline control/controlclient/direct.go:1425]: that means that you won't log context timeout errors, why not?

Comment (maisem) [reply control/controlclient/direct.go:1425]: Also maybe log a ping request so that we can confirm that it made it to the client?

**Review by maisem — APPROVED** [review]

**Review by raggi — APPROVED** [review]
I kind of want to make the wgengine side quack the same interface, but that can be cleaned up later.

**Comment by bradfitz** [comment]
> I kind of want to make the wgengine side quack the same interface, but that can be cleaned up later.

I actually want to kill it. I've been working on removing all its dumb methods: https://github.com/tailscale/tailscale/pull/4754

**Pull request merged**